hw/top_earlgrey/ip/ast/ast_wrapper_pkg.core

```diff
@@ -9,7 +9,6 @@ filesets:
   files_rtl:
     depend:
       - lowrisc:ip:pwrmgr_pkg
```

Comment:
I don't see the pwrmgr package used in `ast_wrapper_pkg.sv`. Can we remove this dependency as well?

Reply:
Done.